src/components/aboutUs/components/TeamProfile.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Avatar, Card, Icon, List } from 'antd';
+
+import { ABOUT_US, TEAM_INFO } from './constants';
+
+const { Meta } = Card;
+const { 
+    CLASSNAMES: { AVATAR, TEAM_CARD, TEAM_PROFILE },
+    STRINGS: { FACEBOOK, GITHUB, ICON, LINKEDIN, SQUARE }, 
+} = ABOUT_US;
+
+class TeamProfile extends React.Component {
+    render() {
+        return(
+            <List  
+                bordered={false}
+                dataSource={TEAM_INFO}
+                grid={{ gutter: 12, lg: 4, md: 4, sm: 1.8, xs: 1.8, xxl: 3 }}
+                renderItem={item => (
+                    <List.Item>
+                        <div key={item.name}>
```

Comment:
no need to have this `div` add the key to the `card` component

Reply:
ok